src/pybind/mgr/mgr_util.py

```diff
@@ -707,50 +692,40 @@ def verify_cacrt(cert_fname):
 def get_cert_issuer_info(crt: str) -> Tuple[Optional[str], Optional[str]]:
     """Basic validation of a ca cert"""
 
-    from OpenSSL import crypto, SSL  # noqa
     try:
-        crt_buffer = crt.encode("ascii") if isinstance(crt, str) else crt
-        (org_name, cn) = (None, None)
-        cert = crypto.load_certificate(crypto.FILETYPE_PEM, crt_buffer)
-        components = cert.get_issuer().get_components()
-        for c in components:
-            if c[0].decode() == 'O':  # org comp
-                org_name = c[1].decode()
-            elif c[0].decode() == 'CN':  # common name comp
-                cn = c[1].decode()
-        return (org_name, cn)
-    except (ValueError, crypto.Error) as e:
+        import subprocess
+        result = subprocess.run(["python3", "-m", "ceph.pybind.mgr.get_cert_issuer_info",
+                                 base64.b64encode(crt if isinstance(crt, bytes) else crt.encode('ascii')).decode("ascii")],
+                                capture_output=True, text=True)
+
+        # Check result with a CompletedProcess
+        if result.returncode != 0 or result.stderr != '':
+            raise ValueError(result.stderr)
+
+        org_name, cn = result.stdout.strip().split("#######")
```

Comment:
Same thing here, 2 different calls, one for getting the org name, and another for the canonical name.

Reply:
Done